### DISCUSSION TITLE
Delete SFT-OOB-LIC before SFT-DCMS-SINGLE?

### DISCUSSION THREAD

**Discussion by pompfe** [zsrv/supermicro-product-key#9] [General]
If some SMC machines already have SFT-OOB-LIC and these machines should now get a SFT-DCMS-SINGLE.  Do we need to delete OOB first (with SUM 2.4.0)?  Or is it ok to set the DCMS key without deletion of OOB?

**Comment by zsrv** [comment]
It's okay to set the DCMS key without deleting the OOB key first.
